sonar-plugin-api/src/main/java/org/sonar/api/utils/DateUtils.java

```diff
@@ -54,6 +54,10 @@ public static String formatDate(Date d) {
     return THREAD_SAFE_DATE_FORMAT.format(d);
   }
 
+  public static String formatDate(long ms) {
```

Comment:
We should try to add API only when needed. Here it's a shortcut, so it's not enough valuable IMO.